From 07bd2542322cc52e482a0016337ef2bcc2e088fb Mon Sep 17 00:00:00 2001
From: "Arun C. Murthy" <acmurthy@apache.org>
Date: Wed, 12 Nov 2014 18:51:46 -0800
Subject: [PATCH 661/791] HDFS-7391. Renable SSLv2Hello in HttpFS. Contributed
 by Robert Kanter.

(cherry picked from commit 9f0319bba1788e4c579ce533b14c0deab63f28ee)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/CHANGES.txt

(cherry picked from commit 7c6f03ba20b9dc8a1b33d5405631c760a904a3bc)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/CHANGES.txt
---
 .../src/main/tomcat/ssl-server.xml                 |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/tomcat/ssl-server.xml b/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/tomcat/ssl-server.xml
index 614b702..c91c2e2 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/tomcat/ssl-server.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/tomcat/ssl-server.xml
@@ -70,7 +70,7 @@
          described in the APR documentation -->
     <Connector port="${httpfs.http.port}" protocol="HTTP/1.1" SSLEnabled="true"
                maxThreads="150" scheme="https" secure="true"
-               clientAuth="false" sslEnabledProtocols="TLSv1"
+               clientAuth="false" sslEnabledProtocols="TLSv1,SSLv2Hello"
                keystoreFile="${httpfs.ssl.keystore.file}"
                keystorePass="${httpfs.ssl.keystore.pass}"/>
 
-- 
1.7.9.5

